### ISSUE TITLE
can't find '../data/online-train.pkl'

### ISSUE THREAD

**Issue by khluu** [JianshuZhang/TAP#6]

**Comment by khluu** [comment]
I found a way to generate data, but now I'm running into an issue where the log reports that it ran out of data.  

```
File "C:\Users\bhoan\Documents\GitHub\TAP\v1\data_iterator.py", line 16, in dataIterator
    features=pkl.load(fp) # load features in dict
EOFError: Ran out of input
```

**Comment by JianshuZhang** [comment]
Looks like you run this code on windows and your file path is wrong, cannot find the file. My path is based on linux, may be you can check your file path again

**Comment by khluu** [comment]
Hi Jianshu,
Looks like there a few files missing in your data folder too? My generator is reporting "on-ascii-test/formulaire004-equation047.ascii not found."?

Thank you,
Kevin

**Comment by khluu** [comment]
I also found a bug (fixed) that this line:
`oupFp_feature=open(outFile,'wr')`
might not work on some platforms since they want 'wb' to write bytes, and pickle wouldn't be able to write down because of this.

**Comment by JianshuZhang** [comment]
Yes, I remember there are one or two files are missing, because source files are wrong.

**Comment by JianshuZhang** [comment]
Yes, good. Python 3 needs ‘wb’.
